### ISSUE TITLE
[Effect Suggestion] Fake Fake Teleport

### ISSUE THREAD

**Issue by Tukat** [gta-chaos-mod/ChaosModV#1218]
**Name of effect**
Fake Fake Teleport

**Effect Description**
Teleports the player to a random location, then teleports them back to their original location for a few seconds, then teleports them back again to the random location.

**Comment by pipecek** [comment]
Cool idea,but, it'd be difficult to code. So difficult infact that I don't think anyone would bother...

**Comment by juhana** [comment]
Why would it be difficult to code?

**Comment by SeanMirrsen** [comment]
Yeah, nothing difficult about several teleports in a row. However the teleportation itself is a bit resource-intensive, so teleporting to and fro multiple times in succession is not the smoothest experience.

That said, I would have liked to see a separate "fakeout fake teleport" effect that, instead of putting the player back where they were, puts them into a different location.

**Comment by TitanY0MaMa** [comment]
Won't that be a little more resource-intensive since the player teleports to a third new location (original, fake and final)?

Because in a Fake Fake Teleport it would switch from original to final to original to final loading the same location over and over again. Am I confusing myself here?

**Comment by juhana** [comment]
There's no problem when there's a delay between the teleports. The game has plenty of time to unload the previous location's resources.
